src/main/drivers/pitotmeter_eagle_tree.c

```diff
@@ -0,0 +1,95 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include <platform.h>
+
+#include "drivers/bus_i2c.h"
+#include "pitotmeter.h"
+#include "drivers/time.h"
+
+#include "common/utils.h"
+#include "common/maths.h"
+
+// eagle_tree, Standard address 0x4D
+#define EAGLE_TREE_ADDR                 0x4D    //the define itself not currently used, but should be used in common_hardware.c and maybe target.c where 0x4D is used
+
+#define EAGLE_TREE_RAW_VALUE_MAX        4048.0f //Max value we can expect from sensor
+#define EAGLE_TREE_RAW_VALUE_MIN        896.0f  //Min value we can expect from the sensor, anything less than this is negative pressure and useless for our needs
+#define EAGLE_TREE_PA_VALUE_MAX         3920.0f //Max value the sensor can read in Pascals
+#define EAGLE_TREE_PA_VALUE_MIN         0.0f    //Min value the sensor can read in Pascals, anything less than this is negative pressure and useless for our needs
+
+
+static uint16_t eagleTreeUp;  // static result of pressure measurement
+static uint8_t rxbuf[4];
+
+static inline float pitotEagleTreeRawToPa(uint16_t rawValue)
+{
+  return (((constrainf((float)rawValue,EAGLE_TREE_RAW_VALUE_MIN,EAGLE_TREE_RAW_VALUE_MAX) - EAGLE_TREE_RAW_VALUE_MIN) * (EAGLE_TREE_PA_VALUE_MAX - EAGLE_TREE_PA_VALUE_MIN)) / (EAGLE_TREE_RAW_VALUE_MAX - EAGLE_TREE_RAW_VALUE_MIN) ) + EAGLE_TREE_PA_VALUE_MIN;
+}
+
+static void eagleTreeStart(pitotDev_t * pitot)
+{
+    busReadBuf( pitot->busDev, 0x00, rxbuf, 2 );
+}
+
+static void eagleTreeRead(pitotDev_t * pitot)
+{
+    if (busReadBuf( pitot->busDev, 0x00, rxbuf, 2 ))
```

Comment:
Coding style - opening curly brace should be on the same line
Coding style - space before/after brackets